### DISCUSSION TITLE
Example to create trace with non-overlapping span & no parent span

### DISCUSSION THREAD

**Discussion by phongngtuan** [open-telemetry/opentelemetry-java#6521] [Q&A]
I've a use case where I want to create a trace with non-overlapping span, each of which is triggered by requests coming from outside the system. There is no well-defined parent span because we don't know in advance how many requests will be sent to this particular entity.

I'm trying to explore using TextMapPropagator like [this example](https://opentelemetry.io/docs/languages/java/instrumentation/#context-propagation-between-http-requests) but having difficulties setting up the initial context so would really appreciate some pointers

**Comment by jkwatson** [comment]
So, let me see if I understand. You want a _trace_ with spans that aren't connected via some sort of root span? That's not something that's possible with most tracing backends. You can have them all have the same trace id, but without a parent, root span, it's going to be tough to stitch them together into a coherent trace.

What backend are you using for displaying/analyzing your traces, and does it support this model?

If you don't have a parent, root span, where are you getting the trace id?
